LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoRaPayloadData.cs

```diff
@@ -293,22 +292,7 @@ public DownlinkPktFwdMessage Serialize(string appSKey, string nwkSKey, string da
             }
 
             SetMic(nwkSKey);
-            var downlinkPktFwdMessage = new DownlinkPktFwdMessage(GetByteMessage(), datr, freq, devEUI, tmst, lnsRxDelay, rfch, time, stationEui);
-            if (Logger.LoggerLevel < LogLevel.Information)
-            {
-                var jsonMsg = JsonConvert.SerializeObject(downlinkPktFwdMessage);
-
-                if (devEUI.Length != 0)
-                {
-                    Logger.Log(devEUI, $"{(LoRaMessageType)Mhdr.Span[0]} {jsonMsg}", LogLevel.Debug);
-                }
-                else
-                {
-                    Logger.Log(ConversionHelper.ByteArrayToString(DevAddr.Span.ToArray()), $"{(LoRaMessageType)Mhdr.Span[0]} {jsonMsg}", LogLevel.Debug);
-                }
-            }
```

Comment:
Intentionally removed?

Reply:
I moved this log statement to the two call-sites and refactored the logic a bit.